common/utils/pom.xml

```diff
@@ -39,6 +39,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
Can we avoid moving this? I like to keep the deps in here as small as possible. Alternatively, please mark the this as a test dependency.